spec/models/services/facebook_spec.rb

```diff
@@ -22,11 +22,39 @@
       post_params = @service.create_post_params(post)
     end
 
-    it 'does not add post link when no photos' do
+    it 'adds URL of post to end if no link at end' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.